stuff/pipelines/send_messages_contest_queue.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+
+'''Send messages to Contest queue in rabbitmq'''
+
+import argparse
+import logging
+import os
+import sys
+import json
+from typing import Optional
+import datetime
```

Comment:
```suggestion
import argparse
import datetime
import logging
import os
import sys
import json

from typing import Optional

```

estos deberían ir en orden lexicográfico

Reply:
Listo